wiki/News_Styling_Criteria/en.md

```diff
@@ -1,12 +1,18 @@
-# News
+# News Styling Criteria
 
-News articles exist on a slightly different system to the main osu!wiki, but follow similar principles. Writing them involves careful attention to a few key features and formats as outlined below.
+*For wiki articles, see: [Article Styling Criteria](/wiki/Article_Styling_Criteria).*
+
+News articles exist on a slightly different system to the main osu! wiki, but follow similar principles. Writing them involves careful attention to a few key features and formats as outlined below.
 
 As always, all news articles should aim to have proper spelling and grammar at a bare minimum, as well as having correct and up-to-date information.
 
 ## Repository directory
 
-The following standards apply only to articles written for the news, and saved in the `news/` directory.
+The following standards apply only to articles written for the news, and saved in the `news` directory.
+
+### News post caching
+
+News posts on the website are cached for up to sixty days before they are fetched again from GitHub. Purging a news post from the cache is possible, but is limited to a few osu! team members. If you had done a quick-fix on a news post that was already posted/merged, tell [Ephemeral](/users/102335) immediately in the `#osu-wiki` [Discord channel](https://discord.gg/ppy) to purge it.
 
 ## Locales
```

Comment:
Probably change this section name? Split it to `British English` and `Locales`. `Locales` will probably have nothing.

Another thing, can you note that news with multiple people writings that it doesn't need to follow the "British English" wording? The Project Loved posts are disobeying that rule.

Reply:
in Project Loved it's like this because the post format makes it very clear that the person is writing a personal message *from America*---it just happens to be that there are multiple writers. I think the "rule" should be that personal thoughts can be written in the author's style, while announcements and the views of osu! should be represented with British English

Reply:
@Joehuu 
> Probably change this section name?

Keeping it as is. This section explains that translations of a news post is not possible, which leads to the British English being required.

---

@cl8n Hmm, is "just use English (but prefer British English)" good enough?